<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>histBackProjectUI</class>
 <widget class="QWidget" name="histBackProjectUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_model_">
       <property name="text">
        <string>model</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_input_">
       <property name="text">
        <string>input</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_model_image_">
       <property name="text">
        <string>model image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_input_image_">
       <property name="text">
        <string>input image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLabel" name="label_opencv_result_image_">
       <property name="text">
        <string>opencv result image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_custom_map_image_">
       <property name="text">
        <string>custom map image</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_opencv_map_image_">
       <property name="text">
        <string>opencv map image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_result_">
       <property name="text">
        <string>custom result</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>opencv result</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>custom map</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>opencv  map</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_custom_result_image_">
       <property name="text">
        <string>custom result image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Morph Kernel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="combobox_morph"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kernel Width</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="slider_kernel_width_">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinbox_kernel_width_">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kernel Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="slider_kernel_height_">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinbox_kernel_height_">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hist Dim</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="slider_hist_dim_">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinbox_hist_dim_">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="button_load_model_">
       <property name="text">
        <string>load</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button_load_input_">
       <property name="text">
        <string>load</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="button_run_">
       <property name="text">
        <string>run</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_kernel_width_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_kernel_width_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_kernel_width_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_kernel_width_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_kernel_height_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_kernel_height_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_kernel_height_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_kernel_height_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_hist_dim_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinbox_hist_dim_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_hist_dim_</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_hist_dim_</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>627</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
